### DISCUSSION TITLE
json_util.py是哪个库的

### DISCUSSION THREAD

**Discussion by Shua-github** [hect0x7/JMComic-Crawler-Python#320] [Q&A] [answered]
我遇到了一个错误，Chatgpt的回复是这样的`错误的根本原因在于 AdvancedDict 类中的 _data 属性没有实现 __deepcopy__ 方法，而 FastAPI 在处理依赖注入时，可能会尝试对该对象进行深拷贝，导致 KeyError。当 copy.deepcopy() 方法试图递归拷贝 AdvancedDict 对象时，它没有找到 __deepcopy__ 方法，进而报错。`
![image](https://github.com/user-attachments/assets/54399b30-77e7-465c-9aac-036aff2331a7)
我想问一下这个文件的仓库/位置在哪，我想pr上去

**Comment by hect0x7** [comment, marked as answer]
https://github.com/hect0x7/common 欢迎pr

**Comment by Shua-github** [comment]
pr了谢谢

**Comment by hect0x7** [comment]
已发布，更新下commonX库即可。
pip install commonX -U
